### ISSUE TITLE
On the relationship with `torchtune`

### ISSUE THREAD

**Issue by jwkirchenbauer** [Lightning-AI/litgpt#1309]
Will it be possible in the future for you to coordinate with the [`torchtune`](https://pytorch.org/blog/torchtune-fine-tune-llms/) project so that we are able to use A for xyz and B for ikj?

We've been using `litgpt` a lot appreciating the nice balance between minimal modeling implementation and key quality of life features for distributed LLM training through the lightning ecosystem.

It seems like since `torchtune`, `lightning`, and `litgpt` are both at varying levels of torch nativeness and the tools for pretraining and finetuning do tend to diverge a bit, it would be cool for us as users of such toolkits if there's an effort to double down on this distinction so that we get optimal utility in all cases without heavy duplication of functionality across libraries.

I already see some nods to `litgpt` in the `torchtune` codebase but I wonder how intentional or open the channels of communication are here.

@carmocca
🤝
@rohan-varma

**Comment by rasbt** [comment]
Glad to hear that you've been enjoying LitGPT

And thanks for bringing that up. What we can tell so far is that torchtune was indeed inspired by litgpt in the motivation and the design (e.g., as quick search here shows: https://github.com/search?q=repo%3Apytorch%2Ftorchtune+lit-gpt&type=pullrequests).

Although we haven't yet heard from the torchtune team or about torchtune itself, we are open to collaboration in the spirit of open source. However, the lack of communication makes it challenging to make any definitive statements at this time.

What we will do on our end is continue to work with the community so that others can join the journey or decide to build on or be inspired by our work.

**Comment by jwkirchenbauer** [comment]
Yep. I performed a similar search of the torchtune repo and that prompted this comment.

Sounds good. I saw a response here a little bit ago, but I will wait for a revised one if there is some news on this.
